Week2task2/src/Name.java

```diff
@@ -0,0 +1,81 @@
+public class Name implements Comparable<Name> {
+	
+	String name;
+	String surname;
+	/**
+	 * @param name
+	 * @param surname
+	 */
+	public Name(String name, String surname) {
+		super();
+		this.name = name;
+		this.surname = surname;
+	}
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public String getSurname() {
+		return surname;
+	}
+
+	public void setSurname(String surname) {
+		this.surname = surname;
+	}
+
+	public Name() {
+		// TODO Auto-generated constructor stub
+	}
+	
+	public Name(Name person) {
+		name = person.name;
+		surname = person.surname;
+	}
+	
+	// check if one Name object is equal to another Name object
+	public boolean equals(Name person1) {
+		if (person1.name.equals(this.name)) {
+			if (person1.surname.equals(this.surname)) {
+				return true;
+			}
+		}
+		return false;
+	}
+	
+	//
+	public int compareTo(Name person1) {
+		int diff = this.name.compareTo(person1.name);
+		int diff2 = this.surname.compareTo(person1.surname);
+		if (diff == 0) {
+			if (diff2 == 0) {
+				return 0;
+			}
+		} else if (diff < 0) {
+				if (diff2 < 0 || diff2 ==0) {
```

Comment:
If you could also please move these if blocks (the one starting at line 58 and the one starting at line 61) to the left a little bit so that they be alligned with the if from the line 54.